tests/runtest.py

```diff
@@ -23,7 +23,8 @@ class TestBase:
     objdir = 'objdir' in os.environ and os.environ['objdir'] or '..'
     uftrace_cmd = objdir + '/uftrace --no-pager -L' + objdir
 
-    default_cflags = ['-fno-inline', '-fno-builtin', '-fno-omit-frame-pointer']
+    default_cflags = ['-fno-inline', '-fno-builtin', \
```

Comment:
I'm ok with this but I guess the backslash at the end of the line is unnecessary though..

Reply:
done